cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/DefaultStoreTest.java

```diff
@@ -113,22 +114,23 @@ public void before() throws Exception {
     admin.create(NamespaceMeta.DEFAULT);
   }
 
-  private void setStartAndRunning(final ProgramRunId id, final long startTime) {
-    setStartAndRunning(id, startTime, ImmutableMap.of(), ImmutableMap.of());
+  private void setStartAndRunning(final ProgramRunId id, final long startTime, ArtifactId artifactId) {
```

Comment:
if all the `artifactId` in this test are the same, maybe can keep a `setStartAndRunning(final ProgramRunId id, final long startTime)` that uses a default `artifactId`

Reply:
they belong to different namespaces in this test class, though the artifact id itself is not validated anywhere, felt its better that they belong to the same namespace as the program we are recording the state for.